ynput/ayon-core
# Blender: Implement 'compress' settings extract blend

## Changelog Description

Implement a toggle in AYON settings to "compress" the `.blend` exports.
It defaults to False, to preserve the old behavior prior to this PR.

## Additional info

Fix #156

## Testing notes:

1. Enable the settings on Extract Blend
2. Export a `.blend` file.
3. The file should be smaller than when the toggle is off.
4. Loading the file should continue to work fine.
